backend/src/main/java/com/bakdata/conquery/models/events/parser/specific/IntegerParser.java

```diff
@@ -87,6 +87,10 @@ protected void registerValue(Long v) {
 			return new RebasingStore(minValue, Integer.MIN_VALUE, IntegerStore.create(getLines()));
 		}
 
+		if(maxValue == Integer.MAX_VALUE){
```

Comment:
```suggestion
		if(maxValue >= Integer.MAX_VALUE){
```
Nur dass es mich beruhigt, weil `maxValue ` ein Long ist

Reply:
Ne, das ist falsch maxValue muss Long.MAX_VALUE sein, dann können wir es nicht abbilden. deswegen auch das todo, hier auf nullbits umzusteigen, falls nötig.

Reply:
Habs behoben